### PR TITLE
Working on Endpoint /users/verify

### PR DESCRIPTION
**Pull request by Deepansharora27**
For #504

### CODE REVIEW

**Comment by Deepansharora27** [comment]
@lakshayman Did you have a look at the PR ?

File: controllers/users.js [hunk 1]
```diff
@@ -8,6 +8,24 @@ const imageService = require('../services/imageService')
  * @param res {Object} - Express response object
  */
 
+
+const verifyUser = async (req, res) => {
+  try {
+    const userId = req.userData.id;
+    await userQuery.addOrUpdate(req.userData.identityStatus, userId);
```
Comment (lakshayman) [inline controllers/users.js:15]:
```suggestion
    await userQuery.addOrUpdate({ identityStatus: 'QUEUED' }, userId);
```

File: controllers/users.js [hunk 2]
```diff
@@ -8,6 +8,24 @@ const imageService = require('../services/imageService')
...
+    let fetchData = {
```
Comment (lakshayman) [inline controllers/users.js:16]:
```suggestion
    let options = {
```

File: controllers/users.js [hunk 3]
```diff
@@ -8,6 +8,24 @@ const imageService = require('../services/imageService')
...
+      body: JSON.stringify({
+        username: req.userData.username,
+        identityURL: req.userData.identityURL,
+      }),
+    };
```
Comment (lakshayman) [inline controllers/users.js:21]: Add method and headers in the options

**Review by lakshayman — CHANGES_REQUESTED** [review]

File: controllers/users.js (hunk 3 above)
Comment (Deepansharora27) [reply controllers/users.js:21]: Done updated the method and headers type

File: controllers/users.js [hunk 4]
```diff
@@ -8,6 +8,28 @@ const imageService = require('../services/imageService')
  * @param res {Object} - Express response object
  */
 
+
+const verifyUser = async (req, res) => {
+  try {
+    const userId = req.userData.id;
+    await userQuery.addOrUpdate(req.userData.identityStatus, userId);
```
Comment (lakshayman) [inline controllers/users.js:15]: requested some changes here to please see that

**Comment by lakshayman** [comment]
Please write the tests too

**Comment by lakshayman** [comment]
Tests are failing for it too, please see that

**Pull request closed without merging**